tests/test_compressed_graph.py

```diff
@@ -653,6 +656,12 @@ def forward(self, x):
 
     TorchBinaryMethodDesc(model_name='embedding', torch_method=F.embedding,
                           input_info=[{"sample_size": [1], "type": "long"}, {"sample_size": [2]}]),
+    TorchBinaryMethodDesc(model_name='embedding_bag', torch_method=F.embedding_bag,
+                          wrap_inputs_fn=partial(n_inputs_fn, nargs=3),
```

Comment:
@vuiseng9, please don't do that. because `TorchBinaryMethodDesc` is supposed to be used for 2 inputs only. But you extend it for an arbitrary number of inputs. There's a `SingleLayerModelDesc` class for it.
Please, just replace it with `SingleLayerModelDesc` here. It should work, as it's a base class and it has all this arguments

Reply:
`SingleLayerModelDesc` for `nn.EmbeddingBag` is added earlier. See line566-567